src/shapes/object.class.js

```diff
@@ -1958,6 +1958,15 @@
    */
   fabric.Object.NUM_FRACTION_DIGITS = 2;
 
+  /**
+   * Defines which properties should be enlivened from the object passed to {@link fabric.Object._fromObject}
+   * @static
+   * @memberOf fabric.Object
+   * @constant
+   * @type string[] 
+   */
+  fabric.Object.ENLIVEN_PROPS = ['clipPath'];
```

Comment:
this is missing in the first PR.

Reply:
Could FILL, STROKE be a part of the enlivable props and handle gradient and patterns?
maybe a tought for a later pr.

As today we have this discrepancy where you can send in the object options a non enlived gradient and pattern and the object will iniitialize them. While this does not work for clipPath.
I wonder if we should make the same for everything

Reply:
Sounds easy to adjust

Reply:
And a good idea

Reply:
> this is missing in the first PR.

I see it on both PRs